pkgs/applications/audio/tuijam/default.nix

```diff
@@ -0,0 +1,46 @@
+{ buildPythonApplication
+, fetchFromGitHub
+, lib
+, python3Packages
+, youtube-dl
+}:
+
+buildPythonApplication rec {
+  pname = "tuijam";
+  version = "unstable-2020-06-05";
+
+  src = fetchFromGitHub {
+    owner = "cfangmeier";
+    repo = pname;
+    rev = "7baec6f6e80ee90da0d0363b430dd7d5695ff03b";
+    sha256 = "1l0s88jvj99jkxnczw5nfj78m8vihh29g815n4mg9jblad23mgx5";
+  };
+
+  buildInputs = [ python3Packages.Babel ];
```

Comment:
this should be `propagatedBuildInputs`, and likely babel wont exist for the application outside of build.